src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -32,3 +32,32 @@ build-themes-Enactors(themes);
 
 // Fonts
 build-fonts-Enactors(fonts);
+
+// OpenDyslexic font-face definitions
+@font-face {
+    font-family: 'OpenDyslexic';
+    src:url('../../../lib/open-dyslexic/fonts/OpenDyslexic-Regular.woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'OpenDyslexic';
+    src:url('../../../lib/open-dyslexic/fonts/OpenDyslexic-Bold.woff');
+    font-weight: 600;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'OpenDyslexic';
+    src:url('../../../lib/open-dyslexic/fonts/OpenDyslexic-Italic.woff');
+    font-weight: normal;
+    font-style: italic;
+}
+
+@font-face {
+    font-family: 'OpenDyslexic';
+    src:url('../../../lib/open-dyslexic/fonts/OpenDyslexic-BoldItalic.woff');
+    font-weight: 600;
+    font-style: italic;
+}
```

Comment:
I wonder if this font-face definition should be part of build-fonts-Enactors method, or at least somehow part of Fonts.styl. It seems that if one were to use that stylus helper but not have manually added the font definition, that it wouldn't work properly.

Reply:
This seems reasonable. It could probably set up as a generator-style function as well (though OpenDyslexic would be the only example).

Reply:
I believe this PR is just waiting on addressing this comment.